docs/guide/assets.md

```diff
@@ -109,7 +109,6 @@ The directory defaults to `<root>/public`, but can be configured via the [`publi
 Note that:
 
 - You should always reference `public` assets using root absolute path - for example, `public/icon.png` should be referenced in source code as `/icon.png`.
```

Comment:
```suggestion
Note that you should always reference `public` assets using root absolute path - for example, `public/icon.png` should be referenced in source code as `/icon.png`.
```

Since it no longer needs to be a list, I think we can combine this as a paragraph.